setup.py

```diff
@@ -4,7 +4,7 @@
 Install PlexAPI
 """
 import re
-from distutils.core import setup
+from setuptools import setup
```

Comment:
Revert this or try exception import error with distutils as fallback. Setuptools isnt part of the stdlib.

Reply:
The python_requires argument needed by Pypi is only supported with setuptools, but not with distutils. So if we fall back on distutils, the keyword could not be specified. Which could of course be done.

However, consider that the Python packaging authority recommends using setuptools over distutils for several years now. See here: https://packaging.python.org/guides/tool-recommendations/#packaging-tool-recommendations

I have not yet seen any pre-installed Python environment on any OS (e.g. Windows, CentOS, Ubuntu, macOS) that does not have setuptools out of the box for years now. All tools creating virtual Python environments install setuptools automatically, and most packages on Pypi depend on setuptools without specifying it in their package requirements.

Let me know whether you still want me to add a fallback to distutils.